src/main/java/bio/terra/cli/service/WorkspaceManagerServiceGcp.java

```diff
@@ -145,6 +157,58 @@ private static GcpAiNotebookInstanceCreationParameters fromCLIObject(
     return notebookParams;
   }
 
+  /**
+   * This method converts this CLI-defined POJO class into the WSM client library-defined request
+   * object.
+   *
+   * @return GCP Dataproc cluster creation parameters in the format expected by the WSM client
+   *     library
+   */
+  private static GcpDataprocClusterCreationParameters fromCLIObject(
```

Comment:
can you put this method in the CreateGcpDataprocClusterParams class and make it a toWsmCreationParams

Reply:
Yeah good idea. Also am curious, why we have this pattern of making an intermediary create/update params obj instead of directly mapping the command class attributes to a wsm create/request body?